src/binpickingtaskzmq.cpp

```diff
@@ -79,23 +79,22 @@ void BinPickingTaskZmqResource::Initialize(const std::string& defaultTaskParamet
     }
 }
 
-void BinPickingTaskZmqResource::ExecuteCommand(const std::string& taskparameters, rapidjson::Document &pt, const double timeout /* [sec] */, const bool getresult)
-{
-    if (!_bIsInitialized) {
-        throw MujinException("BinPicking task is not initialized, please call Initialzie() first.", MEC_Failed);
-    }
-
-    if (!_zmqmujincontrollerclient) {
-        MUJIN_LOG_ERROR("zmqcontrollerclient is not initialized! initialize");
-        _zmqmujincontrollerclient.reset(new ZmqMujinControllerClient(_zmqcontext, _mujinControllerIp, _zmqPort));
+void logTaskParametersAndThrow(const std::string& taskparameters) {
```

Comment:
@felixvd function names are capitalized. i would call it: `_LogTaskParametersAndThrow`

Reply:
pushed